### PR TITLE
Update HowToGetNumbersAndSections.md

### PR DESCRIPTION
**Pull request by pinin4fjords**
Automate some of the release stuff. We can parse the release note files to generate epoch ages and pick the most recent which we can then use to find the studies added since then.

### CODE REVIEW

**Review by pcm32 — APPROVED** [review]
LGTM, thanks @pinin4fjords!

File: sc_helper/bin/start_new_release_notes.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+releaseNotesFile="release-notes/sc/_posts/${releaseDate}-${releaseNumber}.md"
+echo "Will write new release notes to release-notes/sc/_posts/${releaseDate}-${releaseNumber}.md"
+
+if [ -z "$ATLAS_SC_EXPERIMENTS" ]; then
+    echo "You need to set ATLAS_SC_EXPERIMENTS" 1>&2
+    exit 1
+fi
+
+raw_cells=$(printf "%'.f" $(cat $ATLAS_SC_EXPERIMENTS/cell_stats.json | jq '.raw_cells'))
```
Comment (pcm32) [inline sc_helper/bin/start_new_release_notes.sh:19]: Maybe we could push this cell_stats.json every time we load to a hidden location in the FTP, then you don't need to be running on the EBI servers. But we can leave that for later I think.

File: sc_helper/bin/start_new_release_notes.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```
Comment (pcm32) [inline sc_helper/bin/start_new_release_notes.sh:1]: To be more explicit, could you please rename this file to make clear that this is for Single Cell release notes?

Comment (pcm32) [reply sc_helper/bin/start_new_release_notes.sh:1]: ohh, this is inside sc_helper... nevermind.

**Review by pcm32 — APPROVED** [review]
LGTM, great improvement @pinin4fjords

**Comment by pinin4fjords** [comment]
Thanks @pcm32

**Pull request merged**